example/kibana/kibana.yml

```diff
@@ -0,0 +1,13 @@
+---
+## Default Kibana configuration from Kibana base image.
+## https://github.com/elastic/kibana/blob/master/src/dev/build/tasks/os_packages/docker_generator/templates/kibana_yml.template.ts
+#
+server.name: kibana
+server.host: 0.0.0.0
+elasticsearch.hosts: [ "http://elasticsearch:9200" ]
+monitoring.ui.container.elasticsearch.enabled: true
+
+## X-Pack security credentials
+#
+elasticsearch.username: elastic
+elasticsearch.password: '${ELASTIC_PASSWORD}'
```

Comment:
@Rushmead not sure this works in a .conf/.yml? has this change been tested?

Reply:
Yep, tested on our instance that we run, plus the kibana documentation specifies that this is supported :D https://www.elastic.co/guide/en/kibana/current/settings.html
`Finally, environment variables can be injected into configuration using ${MY_ENV_VAR} syntax.`

Reply:
well thats super handy